Sources/Plasma/PubUtilLib/plNetClient/plLinkEffectsMgr.cpp

```diff
@@ -238,30 +238,30 @@ bool plLinkEffectsMgr::MsgReceive(plMessage *msg)
     plNetClientMgr* nc = plNetClientMgr::GetInstance();
     plNetLinkingMgr* lm = plNetLinkingMgr::GetInstance();
 
-    plPseudoLinkEffectMsg* pSeudoMsg = plPseudoLinkEffectMsg::ConvertNoRef(msg);
-    if (pSeudoMsg)
+    plPseudoLinkEffectMsg* pseudoMsg = plPseudoLinkEffectMsg::ConvertNoRef(msg);
+    if (pseudoMsg)
```

Comment:
This actually changes the intended meaning of the variable name from "Plasma Seudo [sic] Message" to "Seudo Message" :wink:

Reply:
Usually the `pSomething` means "pointer to something", not "Plasma something"...

Reply:
Yeah, this was quite likely meant to be cute.  I got the "pointer" prefix, but it just makes it hard to read.